scripts/helmAcs.sh

```diff
@@ -177,7 +193,8 @@ repository:
     initialDelaySeconds: 420
   adminPassword: \"$ALFRESCO_PASSWORD\"
   image:
-    repository: \"alfresco/alfresco-content-repository-aws\"
+    repository: \"$REPO_IMAGE\"
+    tag: \"$REPO_TAG\"
```

Comment:
if condition. if REPO_TAG != "" then tag: \"$REPO_TAG\"
otherwise there will be no tag entry at all